README.md

```diff
@@ -244,7 +238,7 @@ $ mlpack_knn --help
 ```
 
 Running `mlpack_knn` on one dataset (that is, the query and reference
-datasets are the same) and finding the 5 nearest neighbors is very simple:
+datasets are the same) and finding 5 nearest neighbors is very simple:
```

Comment:
The nearest neighbors we are finding aren't approximate, they are exact, so we are talking about *the* 5 nearest neighbors, not just any 5 nearby neighbors.  That's why I specifically added that article.  So I don't think we should drop it.